sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2015,4 +2015,121 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       checkAnswer(table.filter($"p" === "p1\" and q=\"q1").select($"a"), Row(4))
     }
   }
+
+  test("TRIM function-BOTH") {
```

Comment:
Given that we already have tests in sql/SQLQuerySuite, is there any reason we need these tests in hive/SQLQuerySuite? What's the difference?